include/deal.II/lac/la_parallel_vector.templates.h

```diff
@@ -635,6 +720,47 @@ namespace parallel
 
     template <typename Number>
     void
+    Vector<Number>::import(const ReadWriteVector<Number>                  &V,
+                           VectorOperation::values                         operation,
+                           std_cxx11::shared_ptr<const CommunicationPatternBase> communication_pattern)
+    {
+      // If no communication pattern is given, create one. Otherwise, use the
+      // given one.
+      std_cxx11::shared_ptr<const Utilities::MPI::Partitioner> comm_pattern;
```

Comment:
OK. For me, the interface is good :-) I think that deprecating as many old functions as possible (like you did) is the way to go.